### PR TITLE
[FEATURE] added input chunking option in external methods (if these d…

### PR DESCRIPTION
**Pull request by mwalzer**
…o not support huge quantities of input at a time). Also added test

### CODE REVIEW

File: Fred2/test/external/TestExternalEpitopePrediction.py [hunk 1]
```diff
@@ -38,30 +38,54 @@ def test_multiple_inputs(self):
             for v in EpitopePredictorFactory.available_methods()[m]:
                 mo = EpitopePredictorFactory(m, version=v)
                 if isinstance(mo, AExternalEpitopePrediction) and not (mo.version=="0.1" and mo.name=="netmhc"):
-                    if any(a.name in mo.supportedAlleles for a in self.mhcII):
-                        mo.predict(self.peptides_mhcII, alleles=self.mhcII)
-                    else:
-                        mo.predict(self.peptides_mhcI, alleles=self.mhcI)
+                    print "Testing", mo.name, "version", mo.version
```
Comment (b-schubert) [inline Fred2/test/external/TestExternalEpitopePrediction.py:41]: pls delete prints

File: Fred2/test/external/TestExternalEpitopePrediction.py [hunk 2]
```diff
@@ -38,30 +38,54 @@ def test_multiple_inputs(self):
...
     def test_single_epitope_input(self):
         for m in EpitopePredictorFactory.available_methods():
             for v in EpitopePredictorFactory.available_methods()[m]:
                 mo = EpitopePredictorFactory(m, version=v)
                 if isinstance(mo, AExternalEpitopePrediction) and not (mo.version=="0.1" and mo.name=="netmhc"):
-                    if any(a.name in mo.supportedAlleles for a in self.mhcII):
-                        mo.predict(self.peptides_mhcII[0], alleles=self.mhcII)
-                    else:
-                        mo.predict(self.peptides_mhcI[0], alleles=self.mhcI)
+                    print "Testing", mo.name, "version", mo.version
```
Comment (b-schubert) [inline Fred2/test/external/TestExternalEpitopePrediction.py:59]: prints

File: Fred2/test/external/TestExternalEpitopePrediction.py [hunk 3]
```diff
@@ -38,30 +38,54 @@ def test_multiple_inputs(self):
...
     def test_single_allele_input(self):
         for m in EpitopePredictorFactory.available_methods():
             for v in EpitopePredictorFactory.available_methods()[m]:
                 mo = EpitopePredictorFactory(m, version=v)
                 if isinstance(mo, AExternalEpitopePrediction) and not (mo.version=="0.1" and mo.name=="netmhc"):
-                    if any(a.name in mo.supportedAlleles for a in self.mhcII):
-                        mo.predict(self.peptides_mhcII, alleles=self.mhcII[0])
-                    else:
-                        mo.predict(self.peptides_mhcI, alleles=self.mhcI[0])
+                    print "Testing", mo.name, "version", mo.version
```
Comment (b-schubert) [inline Fred2/test/external/TestExternalEpitopePrediction.py:77]: prints

File: Fred2/test/external/TestExternalEpitopePrediction.py (hunk 1 above)
Comment (mwalzer) [inline Fred2/test/external/TestExternalEpitopePrediction.py:41]: no, prints in the test make it instantly clearer which method failed the test  as several methods are tested with this unittest method

File: Fred2/test/external/TestExternalEpitopePrediction.py (hunk 2 above)
Comment (mwalzer) [inline Fred2/test/external/TestExternalEpitopePrediction.py:59]: no, prints in the test make it instantly clearer which method failed the test  as several methods are tested with this unittest method

File: Fred2/test/external/TestExternalEpitopePrediction.py (hunk 3 above)
Comment (mwalzer) [inline Fred2/test/external/TestExternalEpitopePrediction.py:77]: no, prints in the test make it instantly clearer which method failed the test  as several methods are tested with this unittest method

**Pull request merged**